.golangci.yml

```diff
@@ -1,127 +1,132 @@
-run:
-  timeout: 10m
-
+version: "2"
```

Comment:
note: this is auto-migrated by `golangci-lint migrate --config .golangci.yml`, not written by my hand